app/assets/javascripts/libs/render_independently.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import ReactDOM from "react-dom";
+import * as React from "react";
+
+type DestroyFunction = () => void;
+
+export default function renderIndependently(getComponent: DestroyFunction => React$Element<*>) {
```

Comment:
This is very similar to how antd allows calls to `Modal.info` to render a modal in a fire-and-forget-fashion. This is very useful to avoid that our controller has to maintain the timing-dependent state.